docs/README.md

```diff
@@ -1,12 +1,24 @@
 # Kremling
-Kremling is an npm library for writing css with React. It uses [React hooks](https://reactjs.org/hooks), is only 4kb gzipped,
-and requires no changes to your webpack config or build process.
+Kremling is an npm library for writing CSS with React. It is only 4kb gzipped, and requires no changes to your webpack config or build process.
 
 [Kremling Github page](https://github.com/CanopyTax/kremling)
 
 [Walkthrough](/walkthrough/getting-started.md)
 
-## Hooks example
+## Motivation
+Why use Kremling over say, CSS Modules?
+
+1. **Option to use [React hooks](https://reactjs.org/hooks)**. Kremling is bleeding edge. :)
+2. **Readable class names**. In your browser dev tools, `<div class="card">` is easier to understand than `<div class="23fgh_es56_card">`.
+3. **Make it easy to [conditionally apply css](walkthrough/conditional-css.md)**. We can do better than ternaries for changing a DOM element's CSS classes.
+4. **Remove unused CSS from the DOM**. When there are no more components that are using a CSS class, the `<style>` element for those components should be removed from the DOM.
+5. [**Scoped css**](concepts/scoped-css.md). Have CSS rules apply to a component and its children, but no other components. Allow for cascading rules within a "scope".
+
+
+## Hooks
+Hooks are a part of React and Kremling likes them because it makes code extremely functional. (Joel help)
```

Comment:
Why do we need this sentence or the Hooks section?

Reply:
I think it is very unique to Kremling and CSS libraries. From a use POV, I'd like to understand  how it solves a problem. If there is? Or if Kremling thinks it is super cool I think it would be nice to explain a TLDR; about them.

Reply:
It won't be unique about kremling for long 😄. Also, the section right below this section is called Example with Hooks. Is that enough for people to see that we're using hooks?